library/psa_crypto_slot_management.h

```diff
@@ -55,15 +55,18 @@ psa_status_t psa_initialize_key_slots( void );
  * This does not affect persistent storage. */
 void psa_wipe_all_key_slots( void );
 
-/** Allocate a key slot.
+/** Find a free key slot and mark it as in use.
  *
- * \param[out] handle   On success, a handle to a newly allocated key slot.
- *                      0 if an error occurs.
+ * \param[out] handle   On success, a slot number that is not in use. This
```

Comment:
This comments is a bit confusing. Isn't the slot in use after it's marked as in use?

Reply:
I clarified the description.